dev-tools/validation/test_check_dev_demo_summary.py

```
@@ -0,0 +1,745 @@
+            "envFrom:", "missingEnvFrom:", 1
+        )
+        with tempfile.TemporaryDirectory() as directory:
+            root = Path(directory)
+            self._write_workflow_fixture(root, invalid_manifest)
+            with self.assertRaisesRegex(
+                AssertionError,
+                "dev-demo bootstrap pod must import dev-demo-bootstrap-env",
+            ):
+                self.validator.validate_workflow(root)
+
+    def test_smoke_account_runtime_contract_is_redacted_and_bounded(self):
```

Ruff (0.16.0)

[warning] Too many statements (86 > 50)

(PLR0915)